decidim-budgets/app/forms/decidim/budgets/admin/project_form.rb

```diff
@@ -68,20 +67,6 @@ def scope
         def decidim_scope_id
           @decidim_scope_id || scope&.id
         end
-
-        private
-
-        def scope_belongs_to_participatory_space_scope
-          errors.add(:decidim_scope_id, :invalid) if current_participatory_space.out_of_scope?(scope)
-        end
-
-        # This method will add an error to the `attachment` field only if there's
-        # any error in any other field. This is needed because when the form has
-        # an error, the attachment is lost, so we need a way to inform the user of
-        # this problem.
-        def notify_missing_attachment_if_errored
```

Comment:
mm, this validation is not needed anymore?

Reply:
Ooops, I'll review it! 👀 thanks

Reply:
Good catch!! 💪 This was merged recently and missed it, thanks!